bullet_train-themes-light/app/views/showcase/previews/partials/_notices.html.erb

```diff
@@ -0,0 +1,10 @@
+<% showcase.sample "Basic" do %>
+  <% flash[:notice] = "Success Message" %>
+  <% flash[:error] = "Error message" %>
+  <%= render "shared/notices" %>
+<% end %>
+
+
+<% showcase.options do |o| %>
+  <% o.optional :attributes, default: [] %>
+<% end %>
```

Comment:
Looks like the attributes aren't used in the partial, so maybe we should cut them?

```suggestion
```